### PR TITLE
Add 'report Bug' for OpenQA product

### PR DESCRIPTION
**Pull request by DrMullings**
Added a report bug button for the openqa in openqa tests which leads
to progess/openqav3

### CODE REVIEW

File: templates/branding/openSUSE/external_reporting.html.ep [hunk 1]
```diff
@@ -84,7 +86,17 @@ Last good: $last_good (or more recent)
...
 Always latest result in this scenario: [latest]($latest)
 "; %>
-% $product_details{comment} = $description;
+
+% if ($product eq 'OpenQA') {
+%    $product_details{'issue[subject]'} = "test fails in $module";
+%    $product_details{'issue[description]'} = $description;
+% }
+% else {
+% 	 $product_details{short_desc} = "[Build $build] openQA test fails in $module";
```
Comment (okurz) [inline templates/branding/openSUSE/external_reporting.html.ep]: inconsistent indention

File: templates/branding/openSUSE/external_reporting.html.ep [hunk 2]
```diff
@@ -47,8 +47,10 @@
 % elsif ($job->DISTRI eq 'casp') {
 %     $product = $job->VERSION;
 % }
-% $product_details{product} = "$distri $product";
-% $product_details{bug_file_loc} = $step_url;
+% elsif ($job->DISTRI eq 'openqa') {
+%     $product = 'OpenQA';
```
Comment (okurz) [inline templates/branding/openSUSE/external_reporting.html.ep]: s/OpenQA/openQA/

**Comment by DrMullings** [comment]
Fixed indentation and replaced OpenQA with openQA

**Comment by okurz** [comment]
LGTM

**Pull request merged**